requirements/requirements-python3.7.txt

```diff
@@ -372,7 +371,7 @@ virtualenv==20.0.21
 watchtower==0.7.3
 wcwidth==0.1.9
 websocket-client==0.57.0
-wrapt==1.12.1
+wrapt==1.11.2
```

Comment:
Why the downgrade?

Reply:
This is what current pypi and set of dependencies dump when I run --eager update during generate-requirements. Likely triggered by a transient dependency.


As long as they work (they do - all the tests pass) I do not care too much. (and do not have to).  Those are just constraint files (the real requirements and limits are in setup.py). 

We should remember those files are by-product and generated automatically (so they will change pretty much any time setup.py changes and in pretty-much unpredictable ways depending on the transient requirements). This is the same problem we had always, but what we have now is that we capture a "working set of requirements" that not only know can be installed but also that passes all our tests (because those changes can only be merged after all tests pass). 

And this is a set of requirements that today person running 'pip install airflow' will get anyway and we have at least a chance to see if they work. So as long as all tests pass - this is exactly what we need. And I am happy, and frankly - I have no intention to spend any time  chasing which transient requirement caused it.

But feel free to chase it if you really want though :).